pkg/policy/api/rule.go

```diff
@@ -590,3 +633,31 @@ func (kr *PortRuleKafka) GetAPIVersion() (int16, bool) {
 
 	return *kr.apiVersionInt, false
 }
+
+// MapRoleToAPIKey maps the Role to the low level set of APIKeys for that role
+func (kr *PortRuleKafka) MapRoleToAPIKey() bool {
```

Comment:
Please have this return an `error` instead and return error messages specific to why mapping fails

Reply:
will do

Reply:
done